diff --git a/dist/mermaid.core.mjs b/dist/mermaid.core.mjs
index 0a76480239f764f17e1818ac79c1b6c93401a5dc..c4593925a713b88401b4ff345390a3785fa91b88 100644
--- a/dist/mermaid.core.mjs
+++ b/dist/mermaid.core.mjs
@@ -1111,8 +1111,9 @@ var render = /* @__PURE__ */ __name(async function(id32, text, svgContainingElem
   const firstChild = svg.firstChild;
   const diagramClassDefs = diag.renderer.getClasses?.(text, diag);
   const rules = createUserStyles(config, diagramType, diagramClassDefs, idSelector);
+  const rulesDark = createUserStyles({...config, themeVariables: themes_default.dark.getThemeVariables({})}, diagramType, diagramClassDefs, idSelector);
   const style1 = document.createElement("style");
-  style1.innerHTML = rules;
+  style1.innerHTML = rules + `@media (prefers-color-scheme: dark) {${rulesDark}}`;
   svg.insertBefore(style1, firstChild);
   try {
     await diag.renderer.draw(text, id32, "11.14.0", diag);
