diff --git a/dist/mermaid.core.mjs b/dist/mermaid.core.mjs
index 207098ff5f42315d21cd94a3db5b3ed09f1f662d..03b406b00cb9358dac84c338e7c6f6fdff3d8b9c 100644
--- a/dist/mermaid.core.mjs
+++ b/dist/mermaid.core.mjs
@@ -1197,8 +1197,9 @@ var render = /* @__PURE__ */ __name(async function(id33, text, svgContainingElem
   const firstChild = svg.firstChild;
   const diagramClassDefs = diag.renderer.getClasses?.(text, diag);
   const rules = createUserStyles(config, diagramType, diagramClassDefs, idSelector);
+  const rulesDark = createUserStyles({...config, themeVariables: themes_default.dark.getThemeVariables({})}, diagramType, diagramClassDefs, idSelector);
   const style1 = document.createElement("style");
-  style1.innerHTML = rules;
+  style1.innerHTML = rules + `@media (prefers-color-scheme: dark) {${rulesDark}}`;
   svg.insertBefore(style1, firstChild);
   try {
     await diag.renderer.draw(text, id33, "11.15.0", diag);
